[CFD-84] Create Nav Bar component

helenaglow opened the pull request:
### Description
_Sorry I went to sleep so I didnt commit yesterday._


I referenced Jason's PR for the structure of the onPress, onChange, isActive parts.

**Ticket Changes**
- Created two files for the actual nav bar: 
   - `NavTab.tsx`: A tab component for the nav bar, which takes `pageName`, `onPress`, `onChange`, `isActive`, and will also take an icon (did not add that part since waiting for another PR)
   - `NavBar.tsx`: The actual nav bar component. Initializes the tabs and connects to router inside, but I can also make it so that it doesnt do that just let me know 🫡

- Made dummy pages for Calendar, Notices, Forum, and Account in `/pages` folder.

**Unrelated changes:**
- moved `schedule.tsx` and `activity.tsx` into components folder

### Reason for Change
Completes CFD-84


### Type of change
<!--- Please delete options that are not relevant. --->
- [X] New feature (non-breaking change which adds functionality)


### Testing
Add to imports :
```
import NavBar from "../components/NavBar/NavBar";
```

Add to return after `<StatusBar />` :
```
<NavBar />
```

https://github.com/uoftblueprint/centre-for-dreams/assets/113125436/766977b7-6da3-4559-85f1-0097cd9e4933

BakerWJ commented on apps/expo/src/app/pages/account.tsx:
delete the pages directory and make all pages be under the app directory

BakerWJ reviewed (changes requested):
Can you integrate the styling that you have into the native Tabs that is supported by Expo Router -- https://docs.expo.dev/router/advanced/tabs/. Here are the docs for how to change the styling of the tab bar: https://reactnavigation.org/docs/bottom-tab-navigator/#tabbar

You can take a look at the Expo Router docs to see if there is a better option, but I think this is what we want

helenaglow commented:
OK I updated to use expo router! Sorry it took me a minute, the documentation is SO jank. 

I made a `TabRoutes` Constants file but let me know if thats kind of useless. I just thought it could be good to separate.

https://github.com/uoftblueprint/centre-for-dreams/assets/113125436/becdff16-20ff-45b0-b19e-10f066de196a

BakerWJ commented on apps/expo/src/components/NavBar/TabRoutes.tsx:
don't think this is necessary, just use the names in NavBar.tsx

BakerWJ commented on apps/expo/src/app/Account.tsx:
use lowercase for all of the files in the app folder

BakerWJ reviewed (commented):
navbar is looking good, just some small changes

helenaglow commented:
OK, renamed file names and removed the file with the paths 🫡.

Also, because the labels are capitalized, I thought it would be simpler and more easy to understand using `tabBarLabelStyle`.

helenaglow commented:
Yay! I'll go back and fix the icons once the svg thing is set up
